templates/studentas/index.html.twig

```diff
@@ -0,0 +1,40 @@
+{% extends 'base.html.twig' %}
+
+{% block title %}Hello StudentasController!{% endblock %}
+
+{% block body %}
+<style>
+    .example-wrapper { margin: 1em auto; max-width: 800px; width: 95%; font: 18px/1.5 sans-serif; }
+    .example-wrapper code { background: #F5F5F5; padding: 2px 6px; }
+</style>
+<div class="body-bg">
+    <div class="bg-light m-4">
+            <table class="table">
+                <tbody>
+                <tr>
+                    <th scope="row">Projektas</th>
+                    <th scope="row">Studentas</th>
+                </tr>
+                <tr>
+                    <td>{{projectName}}</td>
+                    <td>{{studentName}}</td>
+                </tr>
+                </tbody>
+            </table>
+                        
+                    {% if(studentName=="Matas") %}
```

Comment:
```suggestion
                    {% if studentName == "Matas" %}
```
Skliausteliai Twig'e nėra būtini. Tada kodas būna malonesnis skaityti akiai
(bent jau Symfony oficialioje dokumentacijoje pavyzdžiai yra be skliaustų).